src/run/run_step.go

```diff
@@ -150,46 +150,51 @@ func run(ctx context.Context, state *core.BuildState, label core.AnnotatedOutput
 
 	// ReplaceSequences always quotes stuff in case it contains spaces or special characters,
 	// that works fine if we interpret it as a shell but not to pass it as an argument here.
-	command := ""
 	switch {
+	case overrideCmd != "":
+		command, _ := core.ReplaceSequences(state, target, overrideCmd)
+		// We don't care about passed in args when an override command is provided
+		args = process.BashCommand("bash", strings.Trim(command, "\""), true)
```

Comment:
nit: I know it's a bit silly and probably doesn't matter but we could handle args a bit better here:
```
splitCmd := process.BashCommand("bash", strings.Trim(command, "\""), true)
args = append(splitCmd, args...)
```

Reply:
I've checked and they seem to be ignored either way. For instance running `plz run --cmd="echo 1" //:target arg1 arg2` maps to `bash --noprofile --norc -e -u -o pipefail -c "echo 1" arg1 arg2` which only outputs `1` and ignores the args. Unless you were looking for something else?

Reply:
Ah I see. I suppose the args need to be applies to override command. It's fine, people should just be supplying those sort of things in --cmd.